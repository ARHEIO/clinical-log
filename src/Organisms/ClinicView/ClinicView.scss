.clinic-container {
  h2 {
    margin: 16px;

    @media (min-width: 960px) {
      margin-left: 0;
    }
  }
}
