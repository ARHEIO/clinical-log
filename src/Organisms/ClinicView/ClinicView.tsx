import React, { useEffect, useState, ReactElement } from 'react';
import './ClinicView.scss';
import Spinner from '../../Atoms/Spinner/Spinner';


const ClinicView = (): ReactElement => {
  const [clinicLogs, setClinicLogs] = useState<any | null>(null);

  useEffect(() => {
    setTimeout(() => {
      setClinicLogs({});
    }, 1000);
  }, []);

  return (
    <div className="killer-container">
      <h2>Clinical View</h2>
      { clinicLogs
        ? <div>This is the clinic view</div>
        : <Spinner />}
    </div>
  );
};

export default ClinicView;
