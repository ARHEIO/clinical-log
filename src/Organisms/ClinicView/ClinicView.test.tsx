import React from 'react';
import { render } from '@testing-library/react';
import pretty from 'pretty';
import ClinicView from './ClinicView';

it('renders learn react link', () => {
  const { container } = render(<ClinicView />);
  expect(pretty(container.innerHTML)).toMatchSnapshot();
});
