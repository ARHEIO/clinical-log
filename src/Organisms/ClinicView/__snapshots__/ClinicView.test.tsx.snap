// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clinic view should render 1`] = `
"<div class=\\"killer-container\\">
  <h2>Admin Toolbar</h2>
  <div class=\\"card-container\\">
    <nav class=\\"navigation-toolbar\\"><a class=\\"MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary\\" tabindex=\\"0\\" aria-disabled=\\"false\\" href=\\"#/newsfeed\\"><span class=\\"MuiButton-label\\">Newsfeed</span><span class=\\"MuiTouchRipple-root\\"></span></a><a class=\\"MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary Mui-disabled Mui-disabled\\" tabindex=\\"-1\\" aria-disabled=\\"true\\" href=\\"#/clinic\\"><span class=\\"MuiButton-label\\">Clinical Log</span></a></nav>
  </div>
  <h2>Clinical View</h2>
  <div class=\\"survivor loading-spinner-container\\">
    <div class=\\"loading-spinner\\"><svg class=\\"loading-spinner-rear\\">
        <circle cx=\\"50%\\" cy=\\"50%\\" r=\\"18px\\"></circle>
      </svg><svg class=\\"loading-spinner-front\\">
        <circle cx=\\"50%\\" cy=\\"50%\\" r=\\"18px\\"></circle>
      </svg></div>
  </div>
</div>"
`;
